app/actions/TrezorActions.js

```diff
@@ -993,6 +998,46 @@ export const getWalletCreationMasterPubKey = () => async (
   }
 };
 
+export const TRZ_VOTINGXPRIVSEED_ATTEMPT = "TRZ_VOTINGXPRIVSEED_ATTEMPT";
+export const TRZ_VOTINGXPRIVSEED_FAILED = "TRZ_VOTINGXPRIVSEED_FAILED";
+export const TRZ_VOTINGXPRIVSEED_SUCCESS = "TRZ_VOTINGXPRIVSEED_SUCCESS";
+
+/* eslint-disable no-unused-vars */
+const votingXprivSeed = () => async (dispatch, getState) => {
+  /* eslint-enable no-unused-vars */
+  dispatch({ type: TRZ_VOTINGXPRIVSEED_ATTEMPT });
+
+  if (noDevice(getState)) {
+    dispatch({
+      error: "Device not connected",
+      type: TRZ_VOTINGXPRIVSEED_FAILED
+    });
+    return;
+  }
+
+  const chainParams = selectors.chainParams(getState());
+
+  try {
+    const ckv = await deviceRun(dispatch, getState, async () => {
+      const res = await session.cipherKeyValue({
+        path: accountPath(WALLET_ACCOUNT, chainParams.HDCoinType),
+        key: VOTING_XPRIV_KEY,
+        value: VOTING_XPRIV_VALUE,
+        encrypt: true,
+        askOnEncrypt: true,
+        askOnDecrypt: true
+      });
+      return res.payload;
+    });
+    const seed = Buffer.from(ckv.value, "hex");
+    if (seed.length != 32) throw "unusable voting HD pivate key seed bytes";
+    dispatch({ type: TRZ_VOTINGXPRIVSEED_SUCCESS });
+    return seed;
```

Comment:
I think the best thing to do with the seed is to hand it over to dcrwallet in a new grpc request and have it mathed into the xpriv there, using tested logic.

Reply:
Correct.

Reply:
Will do.